src/ray/common/id.h

```diff
@@ -64,6 +64,9 @@ class BaseID {
   static T FromHex(const std::string &hex_str);
   static const T &Nil();
   static constexpr size_t Size() { return T::Size(); }
+  static bool ValidateBinary(const std::string &binary) {
+    return binary.size() == T::Size() || binary.size() == 0;
```

Comment:
actually why empty is valid?

Reply:
that's what we checked on constructor, not sure if it's really needed